fearless/Modules/Staking/StakingRebondSetup/StakingRebondSetupPresenter.swift

```diff
@@ -53,6 +53,13 @@ final class StakingRebondSetupPresenter {
     }
 
     private func provideAssetViewModel() {
+        if let stakingLedger = stakingLedger,
+           let activeEraInfo = activeEraInfo {
+            unbonding = Decimal.fromSubstrateAmount(
```

Comment:
I think it used in many places and now we have an assumption that this method should be called. I suggest to maybe make this property calculable

Reply:
Fixed